.github/linters/mlc_config.json

```diff
@@ -65,6 +65,9 @@
     },
     {
       "pattern": "^https://aai.egi.eu/device*"
+    },
+    {
+      "pattern": "^../.*/providers/cloud-compute/*"
```

Comment:
why adding this one?

Reply:
Because the link checking is broken since the beginning for this section, due to how the pages are organised, with having multiple markdown files in the same folder, vs having one `_index.md` file per folder.

See https://github.com/EGI-Federation/documentation/tree/main/content/en/providers/cloud-compute/openstack https://github.com/EGI-Federation/documentation/issues/294.

But in fact it seem it's only for `/content/en/providers/cloud-compute/openstack/`, I will update the exclusion.